youtube_dl/extractor/audiomack.py

```diff
@@ -37,33 +38,88 @@ class AudiomackIE(InfoExtractor):
         },
     ]
 
+    @staticmethod
+    def create_song_dictionary(api_response, album_url_tag, track_no=0):
+        # All keys are the same in audiomack api and InfoExtractor format
```

Comment:
This seems to be a really complicated way of writing
```
{
    'title': api_response.get('title', album_url_tag),
    'uploader': api_response.get('artist'),
    'url': api_response['url'],
}
```
Notably, `id` looks really strange - shouldn't that include the `video_id`?